skimage/color/__init__.py

```diff
@@ -68,64 +68,64 @@
 
 
 __all__ = ['convert_colorspace',
-           'guess_spatial_dimensions',
-           'rgba2rgb',
-           'rgb2hsv',
-           'hsv2rgb',
-           'rgb2xyz',
-           'xyz2rgb',
-           'rgb2rgbcie',
-           'rgbcie2rgb',
-           'rgb2grey',
-           'rgb2gray',
-           'gray2rgb',
-           'grey2rgb',
-           'xyz2lab',
-           'lab2xyz',
-           'lab2rgb',
-           'rgb2lab',
-           'rgb2hed',
-           'hed2rgb',
-           'lab2lch',
-           'lch2lab',
-           'rgb2yuv',
-           'yuv2rgb',
-           'rgb2yiq',
-           'yiq2rgb',
-           'rgb2ypbpr',
-           'ypbpr2rgb',
-           'rgb2ycbcr',
-           'ycbcr2rgb',
-           'rgb2ydbdr',
-           'ydbdr2rgb',
-           'separate_stains',
-           'combine_stains',
-           'rgb_from_hed',
-           'hed_from_rgb',
-           'rgb_from_hdx',
-           'hdx_from_rgb',
-           'rgb_from_fgx',
-           'fgx_from_rgb',
-           'rgb_from_bex',
-           'bex_from_rgb',
-           'rgb_from_rbd',
-           'rbd_from_rgb',
-           'rgb_from_gdx',
-           'gdx_from_rgb',
-           'rgb_from_hax',
-           'hax_from_rgb',
-           'rgb_from_bro',
-           'bro_from_rgb',
-           'rgb_from_bpx',
-           'bpx_from_rgb',
-           'rgb_from_ahx',
-           'ahx_from_rgb',
-           'rgb_from_hpx',
-           'hpx_from_rgb',
-           'color_dict',
-           'label2rgb',
-           'deltaE_cie76',
-           'deltaE_ciede94',
-           'deltaE_ciede2000',
-           'deltaE_cmc',
-           ]
+                 'guess_spatial_dimensions',
```

Comment:
It looks like you left an indent while reverting this file. Could you remove it so that this file isn't changed by this PR at all?